config/samples/operator_v1beta1_clustercryostat.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: operator.cryostat.io/v1beta1
+kind: ClusterCryostat
+metadata:
+  name: clustercryostat-sample
+spec:
+  minimal: false
```

Comment:
Should this sample include the deploy and install namespaces spec items?

Reply:
It could, but then I think it would cause the OpenShift Console form view to populate with these values automatically. I'm concerned this might end up being some real namespace and the user might gloss over it, leading to unintended consequences.